src/firefly/plans/util.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+
+__all__ = ["time_converter"]
+
+
+def time_converter(total_seconds):
+    """
+    Convert time (in seconds) to a tuple of hours, minutes, seconds
+
+    Parameters
+    ==========
+    total_seconds (float/int)
+
+    Returns
+    ==========
+    tuple: hours, minutes, seconds
+    """
+
+    # use np.round instead round to prevent errors in round(float("nan"))
+    hours = np.round(total_seconds // 3600)
+    minutes = np.round((total_seconds % 3600) // 60)
+    seconds = np.round(total_seconds % 60)
+    return hours, minutes, seconds
+
+
+def is_valid_value(value):
+    """
+    Check if the value is considered valid for inclusion in metadata.
+    Valid values are non-None, non-false, and if they are collections, they should have a positive length.
```

Comment:
The code does not check that the value is "non-false" (i.e. "true"). Either add this to the code or remove it from the docstring.

Reply:
removed from the docstring; "False" is considered valid, e.g., is_standard: False